Sources/PostgreSQL/Connection.swift

```diff
@@ -1,5 +1,7 @@
 @_exported import SQL
+import Foundation
 import CLibpq
+import CLibvenice
```

Comment:
Does Venice have an API to explicitly clean up a fdwait? We have to wait and then immediately clean it up otherwise when libpq goes to read it causes a crash.